Made Footer annd Navbar

Prizag opened the pull request.

Prizag committed:
Made Footer annd Navbar

advay-nits commented:
First run "pnpm i" once and then push again

Prizag committed:
run pnpmi

Prizag commented:
runned pnpm i

advay-nits commented:
![image](https://github.com/user-attachments/assets/0cbf2295-532d-42c3-a90e-f8cc3c860959)

remove the horizontal scroll-bar

Prizag committed:
Update globals.scss

Removed horizontal scrollbar

Prizag commented:
Removed the horizontal scollbar

advay-nits commented on src/app/components/Navbar.jsx:
```diff
@@ -0,0 +1,118 @@
+              </div>
+            </div>
+            <div className="hidden md:block">
+              <div className="ml-4 flex items-center space-x-4">
```
create an array of objects for navlinks. map over the array to render the links. Also use Link component of Nextjs instead of anchor tag.

advay-nits reviewed (changes requested):
Also decrease the height of the image at footer

advay-nits reviewed (changes requested):
create an array of objects for navlinks. map over the array to render the links. Also use Link component of Nextjs instead of anchor tag.

Prizag committed:
Update Navbar.jsx

created an array of objects for navlinks. maped over the array to render the links. Also used Link component of Nextjs instead of anchor tag.

Prizag commented:
updated Navbar.jsx

vaishali9080 committed:
Update Sidebar.css

Ruler45 committed:
Merge branch 'vaishali9080-patch-1' of https://github.com/advay-nits/Advay-website-nextjs into Footer-and-Navbar

Ruler45 committed:
Final touch to footer and navbar
